### ISSUE TITLE
Overlapping objects in split screen

### ISSUE THREAD

**Issue by narayan954** [bgstaal/multipleWindow3dScene#3]
If we use split screen (feature available in Microsoft edge browser) the object gets duplicated onto the screen (check first screenshot) and when we exit the mode, it becomes one, but strangely (as in second screenshot), and this window gets isolated from other windows and other windows can no longer participate with this window (third screenshot).

![SCREENSHOT 1](https://github.com/bgstaal/multipleWindow3dScene/assets/77617189/879528c1-b06e-42ba-ba9a-20a029657211)
![SCREENSHOT 2](https://github.com/bgstaal/multipleWindow3dScene/assets/77617189/a0427b67-6b9d-4800-99b0-922649cd1dc9)
![SCREENSHOT 3](https://github.com/bgstaal/multipleWindow3dScene/assets/77617189/3c71ca7a-1d6a-4972-b0e7-cb12ce28a1f1)

ps: I'm a beginner here and also trying to understand what and why's that happening,

**Comment by szucsz** [comment]
I think it must be the incognito mode

**Comment by JMcrafter26** [comment]
Try using another browser (it worked in firefox for me)
**AND** disable any adblocker or anti tracking plugin, they will often block this

**Comment by narayan954** [comment]
> Try using another browser (it worked in firefox for me) **AND** disable any adblocker or anti tracking plugin, they will often block this

it works in other browser without this feature(split screen), I'm targeting this feature only

**Comment by narayan954** [comment]
> I think it must be the incognito mode

no, it was just for reference, issue persists in any mode

**Comment by liushengrui527** [comment]
If we use split screen (feature available in Microsoft edge browser) the object gets duplicated onto the screen (check first screenshot) and when we exit the mode, it becomes one, but strangely (as in second screenshot), and this window gets isolated from other windows and other windows can no longer participate with this window (third screenshot).

**Comment by narayan954** [comment]
> If we use split screen (feature available in Microsoft edge browser) the object gets duplicated onto the screen (check first screenshot) and when we exit the mode, it becomes one, but strangely (as in second screenshot), and this window gets isolated from other windows and other windows can no longer participate with this window (third screenshot).

I agree with this, can't we try to fix that?, or we've to create new instance of window each time for this to work? 

Ps, currently only two windows can interact at a time, so if we add a third one, it won't interact with the first two, adding fourth one can work with third, and this pair goes on....
